### PR TITLE
Remove forgotten code in randomMacAddress

### PR DESCRIPTION
**Pull request by Bischoff**
This PR solves issue #795 by removing leftover code.

### CODE REVIEW

**Comment by Bischoff** [comment]
Ping @MalloZup and @Isma399 .

File: libvirt/utils_net.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ const (
 )
 
 // randomMACAddress returns a randomized MAC address
-// with libvirt prefix
+// with QEMU prefix
```
Comment (MalloZup) [inline libvirt/utils_net.go:19]: can we have some documentation or something which can be a comment to refer to this?

**Review by MalloZup — COMMENTED** [review]
@Bischoff  thx for this I would like however that we document better in the code, maybe with an url or doc all this magic prefix somehow

File: libvirt/utils_net.go (hunk 1 above)
Comment (Bischoff) [reply libvirt/utils_net.go:19]: That will probably be hard :-) From what I have seen in the forums, it is a "known fact" but it is hard to find a trusted, official, source.

If you find one, feel free to add it in the comment.

**Comment by Bischoff** [comment]
> @Bischoff thx for this I would like however that we document better in the code, maybe with an url or doc all this magic prefix somehow

I would love that too. So far I failed to find such a thing, but maybe I just missed it... Isma399 , do you have better insights?

**Comment by Isma399** [comment]
No, I have looked for documentation and didn't find anything. 
Neither in http://standards-oui.ieee.org/oui.txt, nor in qemu documentation.

**Comment by Bischoff** [comment]
The, I would suggest just merging like this. This is apparently the kind of facts that everyone uses but that is documented nowhere.

**Review by MalloZup — APPROVED** [review]
I'm ok with this. I'm adding Duncan if he needs more info on this.

thx!

File: libvirt/utils_net.go (hunk 1 above)
Comment (MalloZup) [reply libvirt/utils_net.go:19]: we can put an issue link of this github project. that could be  a doc hint @Bischoff

**Comment by dmacvicar** [comment]
We will skip this for this release. I need some documentation and testcases.

**Comment by MalloZup** [comment]
I think for this PR we need to find out evidence in the codebase https://gitlab.com/libvirt/libvirt and document why we are doing this change and documenting it  + a testcase for this..

Of course previously the behavior was wrong or without technical facts, and  this PR fix it somehow but we need to close the gap and make fix it correctly with a doc comment or link something that justify it more.

Thx @Bischoff and @Isma399  for PR , would be great if one of you could help us to finish this :pray: :sunflower:

**Review by dmacvicar — CHANGES_REQUESTED** [review]
Setting as "Request changes" while tests or documentation is added.

**Comment by Bischoff** [comment]
Test and documentation should have been added to the initial PR that introduced the problem. I won't add them here, it's out of scope of the bug I am trying to fix.

**Pull request closed without merging**